libs/wingc/src/type_check.rs

```diff
@@ -4069,11 +4116,19 @@ impl<'a> TypeChecker<'a> {
 	}
 
 	/// Get's the type of an instance variable in a class
-	fn get_property_from_class_like(&mut self, class: &impl ClassLike, property: &Symbol) -> VariableInfo {
+	fn get_property_from_class_like(
+		&mut self,
+		class: &impl ClassLike,
+		property: &Symbol,
+		allow_static: bool,
```

Comment:
why did you add this?

Reply:
I need to allow calling static methods on Struct definitions, and we cant do the same thing as we do for Json, str, etc.. because the name of the Struct is user defined. (I believe we have special checks for language primitive static functions)

I was not sure how to get around adding this, but would love thoughts.